ansible_collections/f5networks/f5_modules/plugins/modules/bigip_ssl_key_cert.py

```diff
@@ -0,0 +1,815 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2020, F5 Networks Inc.
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'certified'}
+
+DOCUMENTATION = r'''
+---
+module: bigip_ssl_key_cert
+short_description: Import/Delete SSL keys and certs from BIG-IP
+description:
+  - This module will import/delete SSL keys and certs on a BIG-IP.
+    Keys can be imported from key files on the local disk, in PEM format.
+    Certificates can be imported from certificate and key files on the local
+    disk, in PEM format.
+version_added: "f5_modules 1.5.0"
```

Comment:
just use "1.5.0" if the template is not changed we need to change it as this is the correct format now